SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -1579,6 +1620,10 @@ void SiPixelDigitizerAlgorithm::pixel_inefficiency(const PixelEfficiencies& eff,
     
     chips[chipIndex]++;
     columns[dColInDet]++;
+    if (row == 79 || row == 80 || col % 52 == 0 || col % 52 == 51)
```

Comment:
I think this should work for both Phase 0 and 1 if the feature is not turned off for Phase 0. (Although in Phase 0 we had plaquettes where row 80 did not exist.) Again, rowsperroc() and colsperroc() would look better.